lib/modules/datasource/hexpm-bob/index.ts

```diff
@@ -0,0 +1,126 @@
+
+  override readonly caching = true;
+
+  @cache({
+    namespace: `datasource-${datasource}`,
+    key: ({ registryUrl, packageName }: GetReleasesConfig) =>
+      `${registryUrl ?? defaultRegistryUrl}:${packageName}`,
+  })
+  async getReleases({
+    registryUrl,
+    packageName,
+  }: GetReleasesConfig): Promise<ReleaseResult | null> {
+    const packageType = this.getPackageType(packageName);
+
+    if (!packageType) {
+      return null;
+    }
+
+    logger.trace(
+      { registryUrl: registryUrl ?? defaultRegistryUrl, packageName },
+      `fetching hex.pm bob ${packageName} release`
+    );
+
+    const url = `${
+      registryUrl ?? defaultRegistryUrl
+    }/builds/${packageName}/builds.txt`;
+
+    const result: ReleaseResult = {
+      releases: [],
+      ...this.getPackageDetails(packageType),
+    };
+    try {
+      const { body } = await this.http.get(url);
+      result.releases = body
+        .split('\n')
+        .map((line) => line.trim())
+        .filter(is.nonEmptyString)
+        .map((line) => {
+          const [version, gitRef, buildDate] = line.split(' ');
+
+          return {
+            gitRef,
+            isStable: this.isStable(version, packageType),
+            releaseTimestamp: buildDate,
+            version: this.cleanVersion(version, packageType),
+          };
+        });
+    } catch (err) {
+      if (err instanceof HttpError && err.response?.statusCode !== 404) {
+        throw new ExternalHostError(err);
+      }
+      this.handleGenericErrors(err);
+    }
+
+    return result.releases.length > 0 ? result : null;
+  }
+
+  private getPackageType(packageName: string): PackageType | null {
+    if (packageName === 'elixir') {
+      return 'elixir';
+    }
+    if (/^otp\/\w+-\d+\.\d+$/.test(packageName)) {
+      return 'erlang';
+    }
+    return null;
+  }
+
+  private cleanVersion(version: string, packageType: PackageType): string {
+    switch (packageType) {
+      case 'elixir':
+        return version.replace(/^v/, '');
+      case 'erlang':
+        return version.replace(/^OTP-/, '');
+    }
+  }
+
+  private isStable(version: string, packageType: PackageType): boolean {
+    switch (packageType) {
+      case 'elixir':
+        return version.match(/^v\d+\.\d+\.\d+($|-otp)/) !== null;
+      case 'erlang':
+        return version.startsWith('OTP-');
+    }
+  }
+
+  private getPackageDetails(
+    packageType: PackageType
+  ): Omit<ReleaseResult, 'releases'> {
+    let specificDetails: Partial<ReleaseResult>;
+    switch (packageType) {
+      case 'elixir':
+        specificDetails = {
+          homepage: 'https://elixir-lang.org/',
+          sourceUrl: 'https://github.com/elixir-lang/elixir.git',
+        };
+        break;
+      case 'erlang':
+        specificDetails = {
+          homepage: 'https://www.erlang.org/',
+          sourceUrl: 'https://github.com/erlang/otp.git',
+        };
+        break;
+    }
+
+    return { isPrivate: false, ...specificDetails };
```

Comment:
I think `isPrivate: false` is the default (or more specifically `undefined`), so you can omit it.

```suggestion
    return { specificDetails };
```